RELEASE/scripts/autoscend.ash

```diff
@@ -9459,6 +9461,31 @@ boolean L7_crypt()
 		buffMaintain($effect[Soulerskates], 0, 1, 1);
 		buffMaintain($effect[Cletus\'s Canticle of Celerity], 10, 1, 1);
 
+		if (isActuallyEd() && monster_attack($monster[modern zmobie]) >= my_maxhp())
+		{
+			// Need to be able to tank a hit from the modern zmobies as Ed
+			// as we'll never get the jump because their initiative is ridiculous.
+			// Otherwise we'll just die repeatedly.
+			if (get_property("telescopeUpgrades").to_int() > 0 && !get_property("telescopeLookedHigh").to_boolean())
```

Comment:
Yeah it would be nice if we can use something like the maximizer for this or at least some "getBuffs" type function(s). There's a lot of places we could streamline the codebase in that way.